### PR TITLE
Add 50K search term limit

### PR DESCRIPTION
**Pull request by amanifarooque**
## Summary
Updating the limit of query terms we return in the monthly reports. 

Preview: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.sites.pages.cloud.gov/preview/gsa/search-gov-website/update-monthly-report-limit/admin-center/analytics/monthly-reports.html 

## PR Checklist
- [x] Test Cloud.gov Pages preview link for your branch for functionality - make sure all links and buttons work, javascript works as intended, and updates display as expected
- [N/A] Conduct a HemingwayApp review of the content for plain language. Request plain language review in #helpwanted if needed before merging this PR
- [N/A] Conduct a Lighthouse Audit via Chrome Developer Tools. Identify any issues that must be addressed before merging, and fix
- [x] Ensure no non-public information is published in the commit messages

### CODE REVIEW

File: admin-center/analytics/monthly-reports.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ View a detailed report of the number of times searchers have input specific term
 
 Read [Understanding Your Users' Needs By Analyzing Search Terms](https://www.digitalgov.gov/2013/10/24/understanding-your-users-needs-by-analyzing-search-terms/) for tips on how to create a semi-automated report for analyzing the data in this CSV file on a regular basis.
```
Comment (dawnpm) [inline admin-center/analytics/monthly-reports.md:38]: Just noticed - is this link still current?

**Review by dawnpm — APPROVED** [review]

File: admin-center/analytics/monthly-reports.md (hunk 1 above)
Comment (amanifarooque) [reply admin-center/analytics/monthly-reports.md:38]: I think so - it still seems helpful!

Comment (dawnpm) [reply admin-center/analytics/monthly-reports.md:38]: Ok, can you please update the domain so we're not relying on the redirect anymore? many thanks.

**Comment by amanifarooque** [comment]
@dawnpm updated, ready for you to re-approve!

**Comment by amanifarooque** [comment]
@dawnpm Can you re-approve when you get the chance? This change will be deployed today.

**Pull request merged**